content/en/docs/reference/command-line-tools-reference/feature-gates/strict-cos-enforcement-for-webhooks.md

```diff
@@ -0,0 +1,16 @@
+---
+title: StrictCostEnforcementForWebhooks
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
```

Comment:
Similarly, whether this `StrictCostEnforcementForWebhooks` feature gate is in the alpha or beta stage?

	genericfeatures.StrictCostEnforcementForWebhooks: {Default: false, PreRelease: featuregate.Beta},
	
[Referring to kubernetes/kubernetes/pkg/features/kube_features.go#L1302 (here)](https://github.com/kubernetes/kubernetes/blob/6fc0a69044f1ac4c13841ec4391224a2df241460/pkg/features/kube_features.go#L1302)